tetris/index.js

```diff
@@ -0,0 +1,121 @@
+var express = require('express');
+var app = express();
+var http = require('http').Server(app);
+var io = require('socket.io')(http);
+var path = require('path');
+var util = require('util');
```

Comment:
But what if you want to reassign "util" part-way through the program?! Just kidding, good advice.